hack/release.sh

```diff
@@ -42,6 +42,8 @@ OFFICIAL_RELEASE=${OFFICIAL_RELEASE:-false}
 # LOCAL_RELEASE indicates that containerd has been built and released
 # locally.
 LOCAL_RELEASE=${LOCAL_RELEASE:-false}
+GOOS=$(go env GOOS)
```

Comment:
We may get these from elsewhere so let's only set these if not already set..
for example: `GOOS=$(GOOS:-go env GOOS)`

Reply:
I think it's a good idea

Reply:
bash is so much fun sometimes
```
if [ -z "${GOOS}" ]
then
    GOOS=$(go env GOOS)
fi
```